nexus/src/db/queries/external_ip.rs

```diff
@@ -134,8 +135,8 @@ const MAX_PORT: i32 = u16::MAX as _;
 ///     -- possible on replay of a saga node.
 ///     INSERT INTO
 ///         instance_external_ip
-///     (SELECT * FROM external_ip)
-///     ON CONFLICT
+///     (SELECT * FROM next_external_ip)
+///     ON CONFLICT (id)
```

Comment:
I didn't actually change the code here, just updated the documentation of this comment.